site/en/tutorials/images/transfer_learning.ipynb

```diff
@@ -627,7 +627,7 @@
       "source": [
         "### Compile the model\n",
         "\n",
-        "Compile the model before training it. Since there are two classes, use a binary cross-entropy loss with `from_logits=True` since the model provides a linear output."
+        "Compile the model before training it. Since there are two classes, use a `BinaryCrossentropy` loss with `from_logits=True` since the model provides a linear output."
```

Comment:
```suggestion
        "Compile the model before training it. Since there are two classes, use the `tf.keras.losses.BinaryCrossentropy` loss with `from_logits=True` since the model provides a linear output."
```